docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -49,105 +49,1308 @@ SETTINGS cast_keep_nullable = 1
 └──────────────────┴─────────────────────┴──────────────────┘
 ```
 
-## toInt(8\|16\|32\|64\|128\|256)
+## toInt8
 
-Converts an input value to a value the [Int](../data-types/int-uint.md) data type. This function family includes:
+Converts an input value to a value of type `Int8`.
 
-- `toInt8(expr)` — Converts to a value of data type `Int8`.
-- `toInt16(expr)` — Converts to a value of data type `Int16`.
-- `toInt32(expr)` — Converts to a value of data type `Int32`.
-- `toInt64(expr)` — Converts to a value of data type `Int64`.
-- `toInt128(expr)` — Converts to a value of data type `Int128`.
-- `toInt256(expr)` — Converts to a value of data type `Int256`.
+**Syntax**
+
+```sql
+toInt8(expr)
+```
+
+**Arguments**
+
+- `expr` — Expression returning a number or a string with the decimal representation of a number. [Expression](../syntax.md/#syntax-expressions).
+
+:::note
+Binary, octal, and hexadecimal representations of numbers are not supported. Leading zeroes are stripped.
+:::
+
+**Returned value**
+
+- 8-bit integer value. [Int8](../data-types/int-uint.md).
+
+:::note
+Function uses [rounding towards zero](https://en.wikipedia.org/wiki/Rounding#Rounding_towards_zero), meaning it truncates fractional digits of numbers.
+:::
+
+:::danger
+An exception is thrown for [NaN and Inf](../data-types/float.md/#data_type-float-nan-inf) arguments. Keep in mind [numeric conversions issues](#common-issues-with-data-conversion), when using this function.
+:::
+
+**Example**
+
+Query:
+
+```sql
+SELECT
+    toInt8(-8),
+    toInt8(-8.8),
+    toInt8('-8');
+```
+
+Result:
+
+```response
+   ┌─toInt8(-8)─┬─toInt8(-8.8)─┬─toInt8('-8')─┐
+1. │         -8 │           -8 │           -8 │
+   └────────────┴──────────────┴──────────────┘
+```
+
+**See also**
+
+- [`toInt8OrZero`](#toint8orzero).
+- [`toInt8OrNull`](#toint8ornull).
+- [`toInt8OrDefault`](#toint8ordefault).
+
+## toInt8OrZero
+
+Like [`toInt8`](#toint8), it takes an argument of type [String](../data-types/string.md) and tries to parse it to type `Int8`. If unsuccessful, returns `0`.
```

Comment:
`Like toInt8, this function converts an input value to a value of type Int8 but returns `0` in case of an error.